front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -272,17 +272,27 @@ function AssistantBuilderTextArea({
   error?: string | null;
   name: string;
 }) {
+  const textareaRef = useRef<HTMLTextAreaElement>(null);
+
+  useEffect(() => {
```

Comment:
that part seems a bit hacky
Could you check it does what you want, does not flicker, etc.?

Reply:
I have checked yes!